modules/prebidServerBidAdapter/index.js

```diff
@@ -622,7 +623,12 @@ const OPEN_RTB_PROTOCOL = {
           * @type {(string|undefined)}
         */
         if (prop === 'pbadslot') {
-          if (typeof ortb2[prop] === 'string' && ortb2[prop]) utils.deepSetValue(imp, 'ext.data.pbadslot', ortb2[prop]);
+          if (typeof ortb2[prop] === 'string' && ortb2[prop]) {
+            utils.deepSetValue(imp, 'ext.data.pbadslot', ortb2[prop]);
+          } else {
+            // remove pbadslot property if it doesn't meet the spec
+            delete imp.ext.data.pbadslot;
```

Comment:
This feels wrong, but I guess there's a reason why empty string `pbadslot` entries are being filtered.

Reply:
It's being deleted if it's not a string. I don't recall pushing for that validation, but it's ok with me.

Reply:
@bretg @muuki88 -> would it maybe be better to try and convert what is passed to a string first in the else condition and if that doesn't work then we remove the pbadslot property?

Reply:
I think this is ok as-is.